app/views/api/events/index.json.jbuilder

```diff
@@ -0,0 +1,13 @@
+json.events events do |event|
```

Comment:
Here is the jbuilder code.  It's nice to work with, definitely better than active record serialization.  Here is a sample of this response (note the camelCase :)):

```
{"events":[{"id":1,"name":"Thanksgiving","startDate":"2016-11-28","endDate":null,"imageUrl":null,"members":[{"id":1,"name":"Channing","imageUrl":null},{"id":2,"name":"Bryan","imageUrl":null},{"id":3,"name":"Eileen","imageUrl":null}]},{"id":2,"name":"Boulder Visit","startDate":"2016-12-09","endDate":"2016-12-13","imageUrl":null,"members":[{"id":1,"name":"Channing","imageUrl":null},{"id":2,"name":"Bryan","imageUrl":null},{"id":3,"name":"Eileen","imageUrl":null}]}]}